src/tests/utils.ts

```diff
@@ -42,34 +36,24 @@ export async function fundAccount(address: KiltKeyringPair['address'], amount: B
   })
 }
 
-export async function keypairs(account: KiltKeyringPair, mnemonic: string) {
-  const authentication = {
-    ...account.derive('//did//0'),
-    type: 'ed25519',
-  } as KiltKeyringPair
-  const assertion = {
-    ...account.derive('//did//assertion//0'),
-    type: 'ed25519',
-  } as KiltKeyringPair
-  const keyAgreement: KiltEncryptionKeypair = (function () {
-    const secretKeyPair = ed25519PairFromSeed(mnemonicToMiniSecret(mnemonic))
-    const { path } = keyExtractPath('//did//keyAgreement//0')
-    const { secretKey } = keyFromPath(secretKeyPair, path, 'ed25519')
-    return {
-      ...Utils.Crypto.naclBoxPairFromSecret(blake2AsU8a(secretKey)),
-      type: 'x25519',
-    }
-  })()
+export async function keypairs(mnemonic: string) {
+  const authentication = Utils.Crypto.makeKeypairFromUri(mnemonic, signingKeyPairType)
+
+  const assertionMethod = Utils.Crypto.makeKeypairFromUri(mnemonic, signingKeyPairType)
+
+  const keyAgreement = Utils.Crypto.makeEncryptionKeypairFromSeed(Utils.Crypto.mnemonicToMiniSecret(mnemonic))
 
   return {
     authentication,
-    assertion,
+    assertionMethod,
     keyAgreement,
   }
 }
 
 export async function generateDid(account: KiltKeyringPair, mnemonic: string): Promise<DidDocument> {
-  const { authentication, assertion, keyAgreement } = await keypairs(account, mnemonic)
+  ConfigService.get('api')
```

Comment:
what is this about? can you add a comment explaining why we have this line here?

Reply:
I don't think we need this. I can remove it

Reply:
```suggestion
```